.github/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 - 
 
+## [1.7.6]
```

Comment:
your change actually now goes under "unreleased".  you no longer need to pick/assign a number at all.

Reply:
oh thats wicked cool!